src/components/FilterBarInline/filterBarInlineChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Documentation", "Functionality"],
+    notes: [
+      "Adds closeOnBlur prop to the render prop function to apply closeOnBlur to MultiSelect children.",
```

Comment:
This is written differently than the entry in the main CHANGELOG file.  Was that intentional?